src/gleam_stdlib.mjs

```diff
@@ -451,6 +455,13 @@ export function regex_replace(regex, original_string, replacement) {
   return original_string.replaceAll(regex, replacement);
 }
 
+export function regex_replace_first(regex, original_string, replacement) {
+  // Forcibly strip the g flag from the regex, if it's present
```

Comment:
Nit: All JS regexes created in Gleam are global by default so this should always be true. See the `compile_regex` function above.

Reply:
This is why I have to forcibly strip the flag here, and why I confidently assume it will exist instead of checking for it.